styles/base.css

```diff
@@ -18,3 +29,13 @@ a {
   font-family: serif;
```

Comment:
Try some custom fonts, I remember you saying the other day your design always feels a little dated. It's a great way to look more modern! I found this super helpful when picking ones that go together http://www.fontpair.co/